index.d.ts

```diff
@@ -29,6 +29,25 @@ declare const pEachSeries: {
 		iterator: (element: ValueType, index: number) => unknown
 	): Promise<ValueType[]>;
 
+	/**
+	Stop iterating through items by returning this value from the iterator function.
+
+	@example
+	```
+	import pEachSeries = require('p-each-series');
+
+	// Logs "a" and "b"
+	await pEachSeries(['a', 'b', 'c'], value => {
+		console.log(value);
+
+		if (value === 'b') {
+			return pEachSeries.stop;
+		}
+	});
+	```
+	*/
+	readonly stop: unique symbol
```

Comment:
This is not enough. See: https://github.com/sindresorhus/find-up/blob/e852e9cfd020844fac0ea2ff073c378cc302db4d/index.d.ts#L4 You can git blame it to find the PR that added it which has explaination.